src/common/components/Tooltip/Fact/index.js

```diff
@@ -28,13 +36,25 @@ type FactProps = {
     duration: string,
     status: string,
     description: string,
+    apply_count: number,
+    hit_count: number,
+    dmg_multiplier: number,
     prefix: {
       status: string,
       icon: string,
     },
   },
 };
 
+const ATTRIBUTE_MAPPING = {
+  ConditionDamage: 'Condition Damage',
```

Comment:
i tought about running throu translation and then i realized that i don't know all possablities for those mappings..
where can i find all possablities so i can make sure all the values matches with existing translation names?